crawlers/mooncrawl/mooncrawl/data.py

```diff
@@ -67,3 +67,6 @@ class ViewTasks(BaseModel):
     name: str
     outputs: List[Dict[str, Any]]
     address: str
+    customer_id: Optional[str] = None
+    instance_id: Optional[str] = None
+    v3: Optional[bool] = False
```

Comment:
Can we not make it `Optional` or it will break current implementation? Otherwise it is not convenient to have 3 values `None`, `False` and `True`..

Reply:
Yes that are case that it will break current implementation.